admin/weaviate/env_setup.py

```
@@ -0,0 +1,243 @@
+    ATLAS_WEAVIATE_PASSWORD - Atlas service password
+    IRIS_WEAVIATE_USERNAME - Iris service username
+    IRIS_WEAVIATE_PASSWORD - Iris service password
+    ATHENA_WEAVIATE_USERNAME - Athena service username
+    ATHENA_WEAVIATE_PASSWORD - Athena service password
+"""
+
+import os
+import logging
+from typing import Dict, Any
+from dotenv import load_dotenv
+from permissions import WeaviateRBACManager, MICROSERVICE_COLLECTIONS
```

Ruff (0.12.2)

`permissions.MICROSERVICE_COLLECTIONS` imported but unused

Remove unused import: `permissions.MICROSERVICE_COLLECTIONS`

(F401)